### PR TITLE
Setup and initial components

### PR DESCRIPTION
**Pull request by leolpaz**
### Bookstore - initialize the project with components

Added on this pull request:

- React, Redux, and React Router installation.
- Folder structure
- Basic components

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <>
+    <div className="books-container" />
+    <h2>ADD NEW BOOK</h2>
+    <form>
+      <input type="text" placeholder="Book title" />
+      <select disabled>
+        <option value="category">Category</option>
+      </select>
+      <button type="button">ADD BOOK</button>
+    </form>
```
Comment (helman101) [inline src/components/Books.js]: - [x] Please move your form to a separate component,  for example, `CreateNewBook` 😄

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  </>
+);
+
+export default Books;
```
Comment (helman101) [inline src/components/Books.js]: - [x] You should have a component that displays a list of books, and a component that displays a single book, for example `BookList` and `Book`, kindly implement this. 😄

Comment (helman101) [reply src/components/Books.js]: - [x] Your component that lists the books, should pass the data to each book as a prop, for example:
    - `<Book title={book[0]} />`, I would recommend you to create a list of default books on your `Books` state 🚀

**Review by helman101 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🔴 

Hi @leolpaz   👋, 

Nice job so far with this project 😄, but there are still some changes to do.

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by leolpaz** [comment]
Thanks for the review!

**Comment by Davidosky007** [comment]
## Hi @leolpaz 👋, 

## Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
good-job-gif




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**